ethereum_history_api/oracles/src/ethereum/transaction.ts

```diff
@@ -76,7 +76,7 @@ export class TxRlpEncoder {
       ...this.toValueInputToFields(tx),
       this.accessListToFields(tx.accessList),
       encodeField(tx.maxFeePerBlobGas),
-      tx.blobVersionedHashes,
+      [...tx.blobVersionedHashes],
```

Comment:
`blobVersionedHashes` type has changed from `Hex[]` to `readonly Hex[]` so we need to convert to proper type.